frontend_v2/src/app/components/challenge/challengemanage/challengemanage.component.html

```diff
@@ -0,0 +1,24 @@
+<section class="challenge-card ev-sm-container ev-view">
+  <div class="ev-md-container ev-card-panel ev-z-depth-5 ev-challenge-view">
+    <div class="row margin-bottom-cancel">
+      <div class="col s12">
+          <h5 class="w-300">Submission worker actions</h5>
+      </div>
+  </div>
+  <div class="row margin-bottom-cancel">
+      <div class="col s12">
+          <button class="btn ev-btn-dark waves-effect waves-dark grad-btn grad-btn-dark fs-14" (click)="manageWorker('start');"><i class="fa fa-play"></i> Start worker</button>
```

Comment:
This looks redundant code. Can't we have same button for start/stop ?

Reply:
Hey @Shekharrajak I tried having same button for both the start and stop but it is just increasing the  complexity and lines of code (more than 20 lines in place of this single button) as we need `action` param on  button click. If you want to have a look at it, i can push the code.